src/composables/transaction.ts

```diff
@@ -22,33 +32,57 @@ interface ITokenTransactionComposable extends ITx {
   isAe: string
 }
 
-export const useTransactionToken = (
-  initTransaction: ITransaction | undefined,
-  showDetailedAllowanceInfo?: boolean,
-) => {
-  const transaction = ref<ITransaction | undefined>(initTransaction);
+interface UseTransactionOptions {
+  showDetailedAllowanceInfo?: boolean
+}
+
+export function useTransaction({
+  showDetailedAllowanceInfo,
+}: UseTransactionOptions = {}) {
+  const transaction = ref<ITransaction | undefined>();
 
   function setTransaction(newTransaction: ITransaction) {
     transaction.value = newTransaction;
   }
 
-  const fungibleTokens = useState('fungibleTokens');
-  const availableTokens = computed<ITokenList>(() => fungibleTokens.value.availableTokens);
+  const availableTokens = useState('fungibleTokens', 'availableTokens');
```

Comment:
after change from bart, you cant use useState and useGetter in composables @peronczyk